dubbo-remoting/dubbo-remoting-netty4/src/main/java/org/apache/dubbo/remoting/transport/netty4/NettyPortUnificationServerHandler.java

```diff
@@ -49,16 +53,18 @@ public class NettyPortUnificationServerHandler extends ByteToMessageDecoder {
     private final ChannelHandler handler;
     private final boolean detectSsl;
     private final List<WireProtocol> protocols;
+    private Map<String, Channel> dubboChannels;
```

Comment:
The duties of `dubboChannels` and `ChannelGroup` are the same. Need to consider merging.

Reply:
dubboChannels can make pu server manage channels defined in dubbo, and this ablity is needed by some test cases. So I delete DefaultChannelGroup and choose dubboChannels